src/Form/Form.js

```diff
@@ -454,6 +460,7 @@ class Form {
             })
         }
 
+        // We need click coordinates to position the tooltip when the field is in an iframe
```

Comment:
Spent some time with Jonathan to figure out why this stuff was needed and added comments so we know for the future.